Opc.Ua.CloudLib.Client/RestClient.cs

```diff
@@ -141,5 +141,23 @@ private static string PrepareArgumentsString(List<string> arguments)
 
             return stringBuilder.ToString();
         }
+
+        internal async Task<string> UploadNamespaceAsync(UANameSpace nameSpace)
+        {
+            // upload infomodel to cloud library
+            var uploadAddress = client.BaseAddress != null ? new Uri(client.BaseAddress, "infomodel/upload") : null;
+            HttpContent content = new StringContent(JsonConvert.SerializeObject(nameSpace), Encoding.UTF8, "application/json");
+
+            var uploadResponse = await client.SendAsync(new HttpRequestMessage(HttpMethod.Put, uploadAddress) { Content = content }).ConfigureAwait(false);
+            if (uploadResponse != null && uploadResponse.StatusCode != System.Net.HttpStatusCode.OK)
+            {
+                var uploadResponseStr = await uploadResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                return $"{uploadResponse.StatusCode}: Error uploading {nameSpace?.Title}  - {uploadResponseStr}";
+            }
+            else
+            {
+                return null;
```

Comment:
I would return the OK status code on success

Reply:
Returning status and message in either case now.